pandas/tests/series/test_operators.py

```diff
@@ -595,77 +595,103 @@ def test_divide_decimal(self):
 
         assert_series_equal(expected, s)
 
-    def test_div(self):
+    def test_i8_ser_div_i8_ser(self):
```

Comment:
could parametrize on the .astype(np.float32 & np.float64), it doesn't make a real difference, but could check i guess

Reply:
Your comments are all correct about parametrization and redundancy.  This PR got split off of a branch that tried to address all of it at once and got huge because many cases currently fail.  The idea is to split it first in a no-actual-changes-so-easy-to-review way, then apply the fixes one at a time.